warehouse/sentry.py

```diff
@@ -52,6 +48,8 @@ def _sentry(request):
     "gunicorn.http.errors.InvalidProxyLine",
     "gunicorn.http.errors.ForbiddenProxyRequest",
     "gunicorn.http.errors.InvalidSchemeHeaders",
+    # Webob raises this when the client disconnects, which we can't do anything about
+    "webob.request.DisconnectionError",
```

Comment:
It might make sense to import this instead of using a string? I'm fine either way, but the above comment indicates the `gunicorn` errors were only string-ified so we didn't import gunicorn.